mindjunkies/home/tests/test_views.py

```
@@ -0,0 +1,88 @@
+import pytest
+from django.urls import reverse
+from django.contrib.auth import get_user_model
+from model_bakery import baker
+from mindjunkies.courses.models import Course, Enrollment, CourseCategory
```

Ruff (0.8.2)

`mindjunkies.courses.models.CourseCategory` imported but unused

Remove unused import: `mindjunkies.courses.models.CourseCategory`

(F401)